Examples/TodoApp/TodoApp/src/tasks/effecthandlers/TasksListEffectHandler.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+import MobiusCore
+
+final class TasksListEffectHandler: Connectable {
+    weak var viewModel: TasksListViewModeling?
+
+    // MARK: - Connectable
+
+    func connect(_ eventConsumer: @escaping Consumer<TasksList.Event>) -> Connection<TasksList.Effect> {
+        return Connection<TasksList.Effect>(
+            acceptClosure: routeEffect,
+            disposeClosure: { /* Nothing to dispose */ }
+        )
+    }
+
+    // MARK: - Private
+
+    private func routeEffect(_ effect: TasksList.Effect) {
+        switch effect {
+        case .loadTasks:
+            viewModel?.loadTasks()
```

Comment:
I don't understand the indirection to the view model here. I think it would be clearer if you inlined things here.